src/UNL/UCBCN/Calendar.php

```diff
@@ -97,6 +98,10 @@ function links()
                      'uidlastupdated' => 'users:uid');
     }
 
+    public function manageHref() {
```

Comment:
We have been using method names like `getURL()` for stuff like this.  Also, I would return the absolute URL rather than a relative URL because it allows for a wider range of server configurations.  For example, if I install this at `www.example.com/apps/events`, this method would return the a relative URL which would take me to `www.example.com/manager/shortname/manage`, and that URL would no longer be under the `/apps/events/` directory, which would break things.